gossip/src/cluster_info.rs

```diff
@@ -2802,7 +2801,8 @@ impl Node {
                 .expect("tpu_vote multi_bind");
 
         let (tpu_vote_quic_port, tpu_vote_quic) =
-            Self::bind_with_config(bind_ip_addr, port_range, socket_config_reuseport);
+            Self::bind_with_config(bind_ip_addr, port_range, socket_config);
```

Comment:
this causes the tpu-vote-quic to get bound to the same port as repair. we should not allow bind_in_range to accept config with reuseport set.